Training2/src/task1/src/Component/Modal/Modal.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react"
+import './Modal.css'
+
+export default function Modal(){
+    const [modal, setModal] = useState(false)
+
+    const toggleModal = () => {
+      setModal(!modal)
+    }  
+
+    return (
+        <>
+        <button 
+            id='modal__btn' 
+            onClick={toggleModal}
+        >Press me</button>
+    
+        {modal && (
```

Comment:
in the modal concept, we should not unmount whole modal modal inside depending on the toggle variable

the correct way:

- isMounted: this state control the modal inside mounted or not
- isOpenModal: turn on/off modal but just depend on styling, we don't unmount the component once the component mounted
- isMounted = true when initial value of isOpenModal = true or isOpenModal first change to true